keyboards/crkbd/rev1/matrix.c

```diff
@@ -93,6 +93,44 @@ uint8_t matrix_cols(void)
     return MATRIX_COLS;
 }
 
+void tx_rx_leds_init(void)
```

Comment:
@foostan Any issues with these changes?

Reply:
@foostan, @drashna, this is just a way to get access to PD5 and PB0 AtMega32u4 pins. On the regular ProMicro they're hardwired to two TX and RX LEDs, but on Elite-C they're available as general purpose pins, and in my case, I need them to do a matrix scan. 

I believe CrKbd uses them just for visual monitoring of the communication process between two halves. 
Please let me know if there's a better way to do this.

Reply:
I checked it, and no problem.

Reply:
@foostan thanks for verifying this!